recipes/jenkins.rb

```diff
@@ -112,6 +112,24 @@
   end
 end
 
+directory '/var/lib/jenkins/init.groovy.d' do
+  owner 'jenkins'
+  group 'jenkins'
+end
+
+file '/var/lib/jenkins/init.groovy.d/start-quiet.groovy' do
+  content <<-GROOVY
+import jenkins.model.*
+
+Jenkins.get().doQuietDown()
+  GROOVY
+
+  mode '0660'
```

Comment:
Also, we use mode 500: https://github.com/ros-infrastructure/cookbook-ros-buildfarm/pull/147/files#diff-e2bbe1895ea17f8e385d377b5c5b06b4d5720bbce22252f1880269a305304a1aR460

```suggestion
  mode '0500'
```